lib/text-class/text-class.cabal

```diff
@@ -35,7 +35,6 @@ library
     , text
     , time
     , hspec
-    , OddWord
```

Comment:
Hmm. What's the issue with the `OddWord` package?

Reply:
It believe it broke with GHC 9.2. However I realised cardano-node, unlike many of its dependencies, isn't quite supporting GHC 9.2 yet anyway, so it's still 8.10.7 we should be using.

Reply:
Fair enough!

I've added a task to the PR description to "fix OddWord breakage". (Just to remind ourselves.)

Reply:
Already reverted 😉